cypress/components/text-editor/text-editor.cy.js

```diff
@@ -367,6 +370,88 @@ context("Test for TextEditor component", () => {
           });
       });
     });
+
+    describe("rounded corners", () => {
+      it("render with the expected border radius on the toolbar buttons", () => {
```

Comment:
```suggestion
      it("should render with the expected border radius on the toolbar buttons", () => {
```

**nitpick**: grammar really, also consistency with how we've phrased this in similar tests elsewhere. Note: "renders with" (with the S) instead of "should render with" is a valid alternative that we also seem to use in some places.

I only realised after putting this that this isn't actually new, it's just been moved - so while I think it's better changed, don't worry about it!

Reply:
Changed as per comment